CONTRIBUTING.md

```diff
@@ -1,28 +1,100 @@
+This project follows [Google's Open Source Community Guidelines](https://opensource.google/conduct/).
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
+***NOTE***: If you are interested in contributing no-code contributions like user studies, design reviews, UX studies around responsible machine learning, please read the section [How to contribute no-code submissions](#how-to-contribute-no-code-submissions)
 
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+## How to become a contributor and submit your own code
 
-## Code reviews
+### Contributor License Agreements
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
-[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
+We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
 
-## Community Guidelines
+Please fill out either the individual or corporate Contributor License Agreement (CLA).
 
-This project follows [Google's Open Source Community
-Guidelines](https://opensource.google/conduct/).
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](https://code.google.com/legal/individual-cla-v1.0.html).
+  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](https://code.google.com/legal/corporate-cla-v1.0.html).
+
+Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
+
+## Contributing code
+
+If you have improvements to the Model Card Toolkit, send us your pull requests! For those
+just getting started, Github has a [howto](https://help.github.com/articles/using-pull-requests/).
+
+### Code Ownership
+
+* Code ownership is tracked through the `CODEOWNERS` file. Users can be added if one of the following situations apply:
+  * Developers who contribute or maintain the Model Card Toolkit can gain coder owner access to the project.
+  * Requesting project code ownership requires a substantial contribution (e.g. contributed to the bi-weekly meetings, maintained some code over some time, or contributed a major feature).
+
+
+### Development tips
```

Comment:
Let's refactor this to be step-by-step instructions for setting up up a dev environment. We can turn the inline code statements into code blocks for easier copying.